pkg/controller/migmigration/stage.go

```diff
@@ -94,6 +94,12 @@ func BuildStagePods(labels map[string]string,
 func GetApplicationPodsWithStageLabels(labels map[string]string, pvcMapping map[k8sclient.ObjectKey]migapi.PV, list *[]corev1.Pod) []corev1.Pod {
 	applicationPodsWithStageLabel := []corev1.Pod{}
 	for _, pod := range *list {
+		if pod.Status.Phase != corev1.PodRunning {
+			continue
+		}
+		if pod.DeletionTimestamp == nil {
```

Comment:
Isn't this the opposite of what we want? If it's deleted, we skip it. If it's not deleted, we annotate it.

Reply:
Ohh! Good catch!!

Reply:
Good catch Scott.